packages/rich-text/src/to-dom.js

```diff
@@ -207,7 +207,7 @@ export function applyValue( future, current ) {
 				const futureAttributes = futureChild.attributes;
 
 				if ( currentAttributes ) {
-					for ( let ii = 0; ii < currentAttributes.length; ii++ ) {
+					for ( let ii = currentAttributes.length; --ii >= 0; ) {
```

Comment:
Could you explain a bit how the fix works? It looks like you're reversing the loop. Could you add a test case with the same content, but the elements reversed? Wouldn't that fail? Thanks for the PR!